src/rhash/hash.c

```diff
@@ -1859,6 +1970,7 @@ int rc_hash_generate_from_buffer(char hash[33], int console_id, const uint8_t* b
     case RC_CONSOLE_NINTENDO_64:
     case RC_CONSOLE_NINTENDO_DS:
     case RC_CONSOLE_NINTENDO_DSI:
+    case RC_CONSOLE_GAMECUBE:
```

Comment:
Gamecube should not be supported for buffer hashing

Reply:
D'oh! Forgot to remove this, thank you!